jellyfin-helpers/config/plan_workouts.yaml

```diff
@@ -12,6 +12,7 @@ plan_workouts:
     project: Active tasks
     section: ariel_workouts
     task_priority: 2
+    task_priority_optional: 1
```

Comment:
@AkagiShigeru can you verify that this is correct? think the api doesn't match the UI values.
For non-optional ones (default), I want a "blue" coloring &, for optional ones, a "grey" coloring.

Reply:
Yes, that should be correct as specified.

Reply:
Yes, that should be correct as is.